### PR TITLE
Force layout nits

### PR DESCRIPTION
**Pull request by princiya**
In this PR:

- Improved link distance
- Improved tick events (the graph animates slowly onLoad)
- Dynamic node radius based on no. of thirdParties

Gif

### CODE REVIEW

**Comment by princiya** [comment]
@jonathanKingston new PR. let me know what you think :)

File: js/viz.js [hunk 1]
```diff
@@ -60,7 +61,7 @@ const viz = {
 
   stopSimulation() {
     this.simulation.alphaTarget(0);
-    this.simulation.stop();
+    // this.simulation.stop();
```
Comment (jonathanKingston) [inline js/viz.js]: We should likely remove this rather than comment it out.

File: js/viz.js [hunk 2]
```diff
@@ -80,6 +81,7 @@ const viz = {
     this.simulation.force('y', forceY);
 
     const chargeForce = d3.forceManyBody();
+    chargeForce.strength(-100);
```
Comment (jonathanKingston) [inline js/viz.js]: Can you make this a property that has a name that explains what it does?

**Comment by jonathanKingston** [comment]
Thank you!

**Pull request merged**